app/build.gradle.kts

```diff
@@ -73,39 +73,39 @@ dependencies {
 
     /* APS Demand */
     api("com.adsbynimbus.android:extension-aps:$nimbusVersion")
-    api("com.amazon.android:aps-sdk:9.8.1")
+    api("com.amazon.android:aps-sdk:9.10.2")
```

Comment:
Can we comment out these lines but leave them for reference? They override the rich version constraints defined in the SDK and the removal would allow for the preferred versions to be resolved when developing internally.